components/organisms/Education.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {TimelineEvent} from '../molecules/Timeline';
+import {BookOpenIcon} from '@heroicons/react/24/solid';
+
```

Comment:
```suggestion
import { TimelineEvent } from '../molecules/Timeline';
import { BookOpenIcon } from '@heroicons/react/24/solid';
```

Включи ESLint и Prettier в настройках иде